ci/edmtool.py

```diff
@@ -228,7 +228,7 @@ def install(runtime, toolkit, pillow, environment, source):
     # dependencies or we risk testing against an released enable.
     install_local = (
         "edm run -e {environment} -- "
-        "pip install --force-reinstall --no-dependencies " + ROOT
+        "pip install --force-reinstall --no-dependencies -e ."
```

Comment:
This is definitely not the fix we want.  I just pushed this as a sanity check. I still do not exactly understand why the added test fails without this change.

Independent of this though, we probably want to open a separate issue regarding adding an `editable` click option as has been done for other ets projects.  Further, I am not sure why `enable` was using:
`#: Path to the top-level source directory
ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))`
to do this rather than `.` as is done in the other ets projects